app/commands/user/reset_account.rb

```diff
@@ -22,15 +22,25 @@ def call
     end
 
     def reset_tracks!
+      # If someone has left a user track then we won't reset
+      # and things will break. So we need to temporarily recreate
+      # the user tracks.
```

Comment:
Isn't this only an issue when the user has submitted some solutions?

```suggestion
      # If someone has left a user track and submitted some solutions,
      # then we won't reset and things will break. So we need to 
      # temporarily recreate the user tracks.
```